cli/registry.go

```diff
@@ -0,0 +1,254 @@
+package run
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/mitchellh/go-homedir"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+const defaultRegistryContainerName = "k3d-registry"
+
+const defaultRegistryImage = "registry:2"
+
+const defaultIntRegistryPorty = 5000
```

Comment:
I guess the `y` on `RegistryPorty` is unintentional? :grin:

Reply:
In fact, this should be `defaultRegistryPort`.

Reply:
```suggestion
const defaultRegistryPort = 5000
```